lib/PhpStatistics.php

```diff
@@ -64,4 +66,62 @@ protected function cleanVersion($version) {
 		}
 		return $version;
 	}
+
+	/**
+	 * Get status information about the cache from the OPcache extension
+	 *
+	 * @return array with an array of state information about the cache instance
+	 */
+	protected function getOPcacheStatus(): array {
+		// Test if the OPcache module is installed
+		if (!extension_loaded('Zend OPcache')) {
+			// module not loaded, returning back empty array to prevent any errors on JS side.
+			return [];
+		}
+
+		// get status information about the cache
+		$status = opcache_get_status(false);
+
```

Comment:
```
		// Test if the OPcache module is installed
		if (!extension_loaded('Zend OPcache')) {
			// module not loaded, returning back empty array to prevent any errors on JS side.
			return [];
		}

		// get status information about the cache
		$status = opcache_get_status(false);

		if ($status === false) {
			$status = [];
		}

		return $status;
```

I think it's much easier to read like this.